pages/linux/nmcli-connection.md

```diff
@@ -1,7 +1,7 @@
 # nmcli connection
 
 > Connection management with NetworkManager.
-> More information: <https://man.archlinux.org/man/nmcli.1>.
+> More information: <https://manned.org/nmcli>.
```

Comment:
Why not this?
```suggestion
> More information: <https://developer.gnome.org/NetworkManager/stable/nmcli.html>.
```

Reply:
Makes sense — already discussed [above](https://github.com/tldr-pages/tldr/pull/5790#discussion_r615934923). Will change.